features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/messagecomposer/MessageComposerPresenter.kt

```diff
@@ -705,4 +723,33 @@ class MessageComposerPresenter @AssistedInject constructor(
             }
         }
     }
+
+    @OptIn(ExperimentalCoroutinesApi::class)
+    private fun ProduceStateScope<PersistentList<RoomMemberIdentityStateChange>>.observeRoomMemberIdentityStateChange() {
```

Comment:
This method already exists in `IdentityChangeStatePresenter`, we should avoid copy pasting full block of code like this. Can you extract it please?

Reply:
Moved to a Utils class